org.eclipse.winery.repository.ui/src/app/instance/admin/repository/repository.component.ts

```diff
@@ -39,6 +41,21 @@ export class RepositoryComponent implements OnInit {
         );
     }
 
+    checkConsistencyRepository() {
+        this.service.checkConsistencyRepository().subscribe(
+            data => this.handleConsistencyResult(data),
+            error => this.handleError(error)
+        );
+    }
+
+    handleConsistencyResult(data: String) {
+        if (data.length === 0 || isNullOrUndefined(data)) {
```

Comment:
This if statement crashes, doesn't it`?  Should the `isNullOrUndefined` check come first?